capstone-backend/src/main/java/com/google/sps/servlets/UserServlet.java

```diff
@@ -64,13 +68,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         String profileImageUrl = (String) payload.get("picture");
 
         googleUser = new User(googleId, email, fullname, profileImageUrl,
-          new ImmutableMap.Builder<String, String>()
-            .put("access_token", access_token)
-            .put("idpId", idpId)
-            .put("scope", scope)
-            .put("token_id", tokenId)
-            .put("token_type", token_type)
-            .build());
+            new ImmutableMap.Builder<String, String>().put("access_token", access_token).put("idpId", idpId)
+                .put("scope", scope).put("token_id", tokenId).put("token_type", token_type).build());
```

Comment:
nit: change the indentation back to make it more readable.

Reply:
Will do, sorry about this whenever I add and remove CORS lines I use the Gitpod formatter and it changes things like this.